benchmark/buiter_et_al_2008_jgr/figure_6_1e22.prm

```diff
@@ -0,0 +1,189 @@
+set Additional shared libraries = plugin/./libfinite_strain_invariant.so
```

Comment:
If this file is a copy of the other one, just with a modified viscosity, you could use the "include" option of prm files to simplify it. Just start this file with `include figure_6_1e20.prm` and then only add the options that have changed compared to that one (I think you need to keep the `Additional shared libraries` and `Dimension` options in here, because they are needed before the `include` is processed).